src/plots/cartesian/layout_attributes.js

```diff
@@ -680,6 +680,16 @@ module.exports = {
             'If *B*, 1B.'
         ].join(' ')
     },
+    minexponent: {
+        valType: 'number',
+        dflt: 3,
+        min: 0,
+        role: 'style',
+        editType: 'ticks',
+        description: [
+            'Hide SI prefix for 10^n if |n| is below this number'
```

Comment:
Let's add some more info here about when this attribute has an effect.
For example 
```
It has an effect only when the `tickformat` is *SI* or *B*.
```